src/renderers/shaders/UniformsUtils.js

```diff
@@ -86,13 +86,21 @@ export function cloneUniformsGroups( src ) {
 
 export function getUnlitUniformColorSpace( renderer ) {
 
-	if ( renderer.getRenderTarget() === null ) {
+	const currentRenderTarget = renderer.getRenderTarget();
+
+	if ( currentRenderTarget === null ) {
 
 		// https://github.com/mrdoob/three.js/pull/23937#issuecomment-1111067398
 		return renderer.outputColorSpace;
 
 	}
 
+	if ( currentRenderTarget.isXRRenderTarget === true ) {
```

Comment:
```suggestion
	// https://github.com/mrdoob/three.js/issues/27868
	if ( currentRenderTarget.isXRRenderTarget === true ) {
```

Let's include a link to the issue, I think it will be easy to forget why this is a special case. Thanks!